templates/pages/setup/authentication/setup.html.twig

```diff
@@ -0,0 +1,103 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% set session_get_plural = call('Session::getPluralNumber', []) %}
+
+<form name='form' action='{{ action }}' method='post' data-track-changes='true'>
```

Comment:
1. There is a Twig flter to generate form action.
2. We should try to use double quotes in HTML. In the past, PHP default behaviour on some HTML encoding methods was to encode only double quotes and keep single quotes intact. I guess that some libraries may do the same. It seems like an insignificant detail, but can have impacts on security.

```suggestion
<form name="form" action="{{ 'Config'|itemtype_form_path }}" method="post" data-track-changes="true">
```